### DISCUSSION TITLE
What are the underlying models for included detector models?

### DISCUSSION THREAD

**Discussion by fralik** [blakeblackshear/frigate#9007] [Ask A Question] [answered]
From reading the documentation I got the impression that at least some detectors are included as part of frigate. If it is true, what are the underlying models for these detectors?

**Comment by NickM-27** [comment]
all frigate detectors (which are listed at https://docs.frigate.video/frigate/hardware#detectors) come with a default model. They different depending on the detector. They are all based on the COCO dataset

**Reply by fralik** [reply]
Thanks! And what is the architecture? Is it resnet, yolo, simple convnet? Trained on COCO only or fine-tuned on it?

**Reply by NickM-27** [reply, marked as answer]
The arch is generally covered in the docs for the detector. All default models are trained solely on COCO, most models come directly from Google.

coral is MobileDet
https://github.com/blakeblackshear/frigate/blob/c35c7da82a23520478ca56dd469fc28dcf1e1e5a/docker/main/Dockerfile#L96

openvino is MobileNet https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/ssdlite_mobilenet_v2

and TensorRT / RKNN are yolo

**Reply by svenstaro** [reply]
I think it might be good idea to mention the default model for Coral in the docs, too. Currently it's only mentioned for OpenVINO.
